input/docs/cs/imagenet_classes.md

```diff
@@ -0,0 +1,22 @@
+title: Zobrazení tříd neuronových konvolučních sítí
+Description: Zobrazení tříd z konvoluční sitě Alexnet v čitelné podobě
+---
+>Tento dokument pracuje s konvoluční neuronovou sítí Alexnet.
+
+#Proč si vytvořit jednoduchý skript pro zobrazení klasifikací
```

Comment:
Bez podnadpisů, pokud bude dobře název a Description, tak toto není nutné. Stejně jako většina textu pod tím. Zkus to trochu zestručnit.